apps/frontend/src/app/Data/Characters/RaidenShogun/index.tsx

```diff
@@ -277,6 +262,7 @@ export const data = dataObjForCharacterSheet(
       burstBoost: nodeC3,
       burst_dmg_: skillEye_,
       electro_dmg_: dmgFormulas.passive2.passive2ElecDmgBonus,
+      enemyDefIgn_: defIgn_
```

Comment:
probably needs to be premod: { enemy: defIgn_ }

Reply:
seems like it's indeed that, just defIgn rather than defIgn_